app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,17 @@
+class CommentsController < ApplicationController
+
+  def create
+    @comment = Comment.new(comment_params)
+    @comment.job_id = params[:job_id]
+    @comment.save
+
```

Comment:
I like this, i think another way to provide clarity for which job is to say `@job = Job.find(params[:job_id])` then `@comment = job.comments.create(comment_params)`